packages/site/cms-site/src/sitePreview/SitePreviewUtils.ts

```diff
@@ -25,7 +26,7 @@ function getPreviewScopeSigningKey() {
     return process.env.SITE_PREVIEW_SECRET || "secret";
 }
 
-export async function sitePreviewRoute(request: NextRequest, graphQLFetch: GraphQLFetch) {
+export async function sitePreviewRoute(request: NextRequest, graphQLFetch: GraphQLFetch, options?: { apiRoutePostfix?: string }) {
```

Comment:
To be correct, the opposite of prefix is suffix: https://twominenglish.com/postfix-vs-suffix/ 😁

Reply:
😬